src/locales/en-US/admin.ftl

```diff
@@ -607,3 +607,10 @@ invite-goToAdmin = Go to { -product-name } Admin
 invite-goToOrganization = Go to { $organizationName }
 invite-tokenNotFound =
   The specified link is invalid, check to see if it was copied correctly.
+
+userDetails-banned-on = <strong>Banned on</strong> { $timestamp }
+userDetails-banned-by = <strong>by</strong> { $username } 
+userDetails-suspension = Suspension 
+userDetails-suspended-by = <strong>by</strong> { $username }
```

Comment:
Can we join together the two element `Suspension` and `by { $username }`?

The reason being that it's hard to translate two separate ordered items in a sentence. In other languages (say Japanese) the subject of the sentence `{ $username }` will come before the word "Suspension".

By creating two elements, we make it hard for them to reorder this translation and template it nicely.

Reply:
i had them separate because "suspension" is on a different line (it's more like the heading of the bubble). Could I wrap two separate `<Typography>` elements in the same `<Localization>` tag?

Reply:
I'm not sure if we can wrap two of them together like that. In that case I think this is probably fine?